code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -182,9 +182,12 @@
 
 /obj/item/projectile/bullet/sniper/accelerator/Range()
 	..()
-	damage += 5
-	if(damage > 40)
-		icon_state = "gaussstrong"
+	damage += 7
+	transform *= TransformUsingVariable(1.2 , 100)
+	if(damage > 42)
 		breakthings = TRUE
-	else if(damage > 25)
-		icon_state = "gauss"
+
+/obj/item/projectile/bullet/sniper/accelerator/laser
```

Comment:
Agreed. You would not have to check for so many things in your testing this way. Either you are going to be copy-pasting the bullet's unique code to the laser, or copy-pasting the laser's properties to a projectile. It is better to keep the subtypes consistent in this case.

Of course, if you were ultra obsessive, your third option is to elevate the accelerator properties such that they can be used by both physical and energy projectiles. I do not recommend it, however.